functions/menu.bash

```diff
@@ -136,13 +136,15 @@ show_main_menu() {
     "   | Disable NTP"            "Disable time synchronization via systemd-timesyncd to NTP servers" \
     "34 | Change passwords"       "Change passwords for Samba, openHAB Console or the system user" \
     "35 | Serial port"            "Prepare serial ports for peripherals like Razberry, SCC, Pine64 ZWave, ..." \
-    "36 | WiFi setup"             "Configure wireless network connection" \
+    "36 | Disable framebuffer"    "Disable framebuffer on RPi to minimize memory usage" \
```

Comment:
You need to make sure to expand the menu size to accommodate the new menu options.

Reply:
done